src/src/PluginStructs/P095_data_struct.cpp

```diff
@@ -42,6 +42,16 @@ P095_data_struct::P095_data_struct(uint8_t             rotation,
   _commandTriggerCmd += F("cmd");
 }
 
+/****************************************************************************
+ * Destructor
+ ***************************************************************************/
+P095_data_struct::~P095_data_struct() {
+  if (nullptr != tft) {
+    delete tft;
+    tft = nullptr;
+  }
```

Comment:
What about `gfxHelper` ?

Reply:
Ah, I copied that from PR #3903 but there the `gfxHelper` doesn't exist yet 😉

Reply:
In here you know the actual type of the base pointer, so you could downcast it to the actual type and call the destructor of that one. 
But fixing the virtual destructor in the base class is the best anyway :)